src/test/scala/mesosphere/marathon/api/v2/ModelValidationTest.scala

```diff
@@ -103,7 +103,7 @@ class ModelValidationTest
       case f: Failure =>
         val errors = (Json.toJson(f) \ "details").as[Seq[JsObject]]
         errors should have size 1
-        (errors.head \ "path").as[String] should be("/groups(0)/apps(1)")
+        (errors.head \ "path").as[String] should be("/groups(group1)/apps(invalid)")
```

Comment:
/cc @pierlo-upitup @aquamatthias Absolute paths in the parenthesis seem redundant and hard to read that way. That said if you have a validation error for a single app, they give you context:

`/app(/nested/group/app1)/cmd`

instead of

`/app(app1)/cmd`